examples/17_resnet34_imagenet_conversion_to_analog.py

```diff
@@ -21,25 +21,30 @@
 from torchvision.models import resnet34
 
 # Imports from aihwkit.
-from aihwkit.nn.conversion import convert_to_analog
+from aihwkit.nn.conversion import convert_to_analog_mapped
 from aihwkit.simulator.presets import TikiTakaReRamSBPreset
+from aihwkit.simulator.configs.utils import MappingParameter
 
-# Device used in the RPU tile
-RPU_CONFIG = TikiTakaReRamSBPreset()
+# Example: Load a predefined model from pytorch library and convert to
+#          its analog version.
 
+# Load a pytorch model.
+model = resnet34()
+print(model)
 
-def main():
-    """Load a predefined model from pytorch library and convert to its analog version."""
-    # Load the pytorch model.
-    model = resnet34()
-    print(model)
+# Define device and chip configuration used in the RPU tile
+mapping = MappingParameter(max_input_size=512,  # analog tile size
+                           max_output_size=512,
+                           digital_bias=True)  # whether to use analog or digital bias
+# choose any preset or RPU configuration or preset, here with custom mapping
```

Comment:
preset is redundant in the comment : Choose any preset of RPU configuration with custom mapping.